website/docs/r/data_lake_store_file.html.markdown

```diff
@@ -43,3 +43,11 @@ The following arguments are supported:
 * `local_file_path` - (Required) The path to the local file to be added to the Data Lake Store.
 
 * `remote_file_path` - (Required) The path created for the file on the Data Lake Store.
+
+## Import
```

Comment:
It looks like this PR is missing the importer block and corresponding test

Reply:
yes, let me push that 🤦‍♂️